generator/C/include_v2.0/mavlink_helpers.h

```diff
@@ -92,9 +92,11 @@ MAVLINK_HELPER uint8_t mavlink_sign_packet(mavlink_signing_t *signing,
 }
 
 /**
- * return new packet length for trimming payload of any trailing zero
- * bytes. Used in MAVLink2 to give simple support for variable length
- * arrays.
+ * @brief Trim payload of any trailing zero-populated bytes (MAVLink 2 only).
+ *
+ * @param payload Serialised payload buffer.
+ * @param length Length of full-width payload buffer.
+ * @return Length of payload after empty bytes trimmed.
```

Comment:
Should his be `zero bytes trimmed`?

Reply:
Yes. Changed to zero-filled bytes. Thanks